content/create-new-collection.en.md

```diff
@@ -0,0 +1,159 @@
+- Enable “Allow auto-merge”.
+- Enable “Automatically delete head branches”.
+- In the `Branches` section: add a branch protection rule for `main`, check “Require status checks to pass before merging” and add `validate_modified_declarations` and `validate_schema` as required status checks.
+- In the `Actions` section, in the `General` submenu: select "Allow all actions and reusable workflows".
+
+#### Update readme
+
+- Update readme with proper metadata: topic, maintainers, jurisdictions, languages…
+
+#### Update deployment inventory
+
+- Open the `deployment/inventory.yml` and set server information: ip, user, fingerprint…
+
+### Snapshots
+
+Create the snapshot repository by using the [Demo snapshots](https://github.com/OpenTermsArchive/demo-snapshots) as template:
+- Go to the [demo snapshots repository homepage](https://github.com/OpenTermsArchive/demo-snapshots)
+- Click on the "Use this template" dropdown
+- Select "Create a new repository"
+- In the opened page, fill the repository name, it is strongly recommended to use the `<collection_id>-snapshots`.
+- When redirected to the newly generated repositroy, please wait until the automatic setup is done (it should be done in less than a minute).
+- Give write rights on the repo to the maintenance team.
+
+#### Fill the about section
+
+- Set the description: “Documents snapshots for `<collection_name>`. Maintained by `<maintainer>`.”
+- Set website: https://opentermsarchive.org
+- Add the following tags: `terms-of-service`, `terms-of-service-agreements`, `terms-and-conditions`, `open-terms-archive`.
+- Uncheck “Releases”, “Packages” and “Environments”.
+
+#### Define repository settings
+
+These settings aim at minimising the otherwise overwhelming amount of information and click targets.
+
+- Features: uncheck “Wikis” and “Projects”.
+- In the `Actions` section, in the `General` submenu: select "Disable actions".
+
+### Versions
+
+Create the snapshot repository by using the [Demo versions](https://github.com/OpenTermsArchive/demo-versions) as template:
+- Go to the [demo versions repository homepage](https://github.com/OpenTermsArchive/demo-versions)
+- Click on the "Use this template" dropdown
+- Select "Create a new repository"
+- In the opened page, fill the repository name, it is strongly recommended to use the `<collection_id>-versions`.
+- When redirected to the newly generated repositroy, please wait until the automatic setup is done (it should be done in less than a minute).
+- Give write rights on the repo to the maintenance team.
+
+#### Fill the about section
+
+- Set the description: “Documents versions for `<collection_name>`. Maintained by `<maintainer>`.”
+- Set website: https://docs.opentermsarchive.org/navigate-history/
+- Add the following tags: `terms-of-service`, `terms-of-service-agreements`, `terms-and-conditions`, `open-terms-archive`.
+- Uncheck “Packages” and “Environments”.
```

Comment:
```suggestion
- [] Uncheck “Packages” and “Environments”.
```

I wonder if it wouldn't be better to have checkboxes, which would make it easier to use this list as a list of things done / not yet done. What do you think ?

Reply:
I would like to have a checklist too, but since this is a documentation page, I don't see how this checklist would be actionable.